core/src/main/java/com/google/errorprone/bugpatterns/MustBeClosedChecker.java

```diff
@@ -125,6 +125,15 @@ public Description matchNewClass(NewClassTree tree, VisitorState state) {
     return matchNewClassOrMethodInvocation(tree, state);
   }
 
+  @Override
+  protected Description matchNewClassOrMethodInvocation(ExpressionTree tree, VisitorState state) {
+    Description description = super.matchNewClassOrMethodInvocation(tree, state);
+    if (description.equals(NO_MATCH)) {
+      return NO_MATCH;
+    }
+    return description;
+  }
```

Comment:
:eyes: This override can be dropped, no?

(Or does the Google-internal code base have some extra code after the `if`-statement?)